azext_iot/tests/__init__.py

```diff
@@ -86,6 +98,210 @@ def command_execute_assert(self, command, asserts):
 
         return output
 
+    def _create_device(self, **kwargs) -> (str, str):
```

Comment:
Continuing on [this comment](https://github.com/Azure/azure-iot-cli-extension/pull/403/files#r687213101), Central test helpers should be organized under `tests/central/*` like how digitaltwins tests and related parts are organized.

Reply:
`CaptureOutputLiveScenarioTest` is meant to be a generic test class for capturing output, it is not meant to encapsulate iot central test helpers.